### PR TITLE
Paw dev

### PR DESCRIPTION
**Pull request by kokoaono**
added tests for Pets component to render button, link and images correctly.

### CODE REVIEW

File: src/components/pets/__tests__/pets.test.js [hunk 1]
```diff
@@ -25,3 +25,36 @@ test("matches animal link snapshot", () => {
...
+test("Pets component should render with button", () => {
+  render(
+    <BrowserRouter>
+      <Pets />
+    </BrowserRouter>
+  );
+  const ClickHereButton = screen.getAllByRole("button", {
+    name: /Click Here/i,
+  });
+  expect(ClickHereButton[0]).toHaveAttribute("href", "/dog");
```
Comment (redxzeta) [inline src/components/pets/__tests__/pets.test.js:38]: dont forget to check cat bird rabbit

File: src/components/pets/__tests__/pets.test.js [hunk 2]
```diff
@@ -25,3 +25,36 @@ test("matches animal link snapshot", () => {
...
+
+test("should render list", () => {
+  render(
+    <BrowserRouter>
+      <Pets />
+    </BrowserRouter>
+  );
+  const petTypeList = screen.getAllByRole("link");
+  expect(petTypeList.length).toBe(5);
+  expect(petTypeList[0]).toHaveAttribute("href", "/dog");
```
Comment (redxzeta) [inline src/components/pets/__tests__/pets.test.js:49]: dont forget to check cat bird rabbit

**Review by redxzeta — CHANGES_REQUESTED** [review]
Good start so far 

after making the changes, 
check out if you can test the onHover using `userEvent.hover` and `userEvent.unhover`

[hover and unhover](https://testing-library.com/docs/ecosystem-user-event#hoverelement-options)


maybe look into mocking `linkData` with 3 items or something and test it out

File: src/components/pets/__tests__/pets.test.js (hunk 2 above)
Comment (kokoaono) [reply src/components/pets/__tests__/pets.test.js:49]: will do.

**Review by redxzeta — APPROVED** [review]

**Pull request merged**